### PR TITLE
feat: add support for windows and macos jobs

### LINKED ISSUE #16: Other operating systems support
What dependency this is using that prevents it from running on other operating systems?

### PR DESCRIPTION
**Pull request by Ameausoone**
BREAKING CHANGE: The action implementation move from container action to nodejs action

Closes #16

### CODE REVIEW

**Comment by rlespinasse** [comment]
Due to auto-close of the #21. This PR have been created

File: __tests__/shortsha.test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {shortsha} from '../src/slug'
+
+function test_short_sha(input: string, expected: string) {
```
Comment (rlespinasse) [inline __tests__/shortsha.test.ts:3]: https://github.com/rlespinasse/github-slug-action/pull/21#discussion_r482905369 from @ChristianUlbrich

Comment (Ameausoone) [reply __tests__/shortsha.test.ts:3]: I let you choose what you prefer @rlespinasse.

**Review by rlespinasse — APPROVED** [review]

**Pull request merged**